etstool.py

```diff
@@ -151,6 +151,7 @@ def cli():
 @click.option("--toolkit", default="pyqt", help="Toolkit and API to use")
 @click.option("--environment", default=None, help="EDM environment to use")
 @click.option('--source/--no-source', default=False)
+@click.option('--quiet/--no-quiet', default=True)
 def install(edm, runtime, toolkit, environment, source):
```

Comment:
We're missing a `quiet` argument in the `install` function/command. Ref : https://click.palletsprojects.com/en/7.x/ click is the library that we use to enable parsing the command line arguments.